src/os2datascanner/projects/admin/tests/test_miniscanner.py

```diff
@@ -0,0 +1,135 @@
+from os2datascanner.projects.admin.adminapp.views.miniscanner_views import *
+from random import choice
+
+class DummyFile():
+    def __init__(self, name, fileContents) -> None:
+        self.contents = fileContents
+        self.name = name
+
+    def read(self):
+        return self.contents.encode()
+
+class FilesObj():
+    def __init__(self, objs) -> None:
+        self.files = objs
+    def get(self, obj):
+        return self.files[obj]
+    
+class Post():
+    def __init__(self, objs) -> None:
+        self.items = objs
+    def get(self, obj):
+        return self.items[obj]
+    
+class Meta():
+    def __init__(self, objs) -> None:
+        self.items = objs
+    def get(self, obj):
+        return self.items[obj]
```

Comment:
These three objects appear to be identical apart from the name of their internal collection. What's up with that?

Reply:
The execute_mini_scan expects a Request obj, and calls request.FILES.get(whatever), request.META.get(), request.POST.get(). Now that you point it out, i could make them inherit from a bigger class, since they are identical, if that's what you mean.

Reply:
They all know inherit from a bigger class. Let me know if you had something else in mind for the fix.

Reply:
This looks better now, thanks, but... why have the three subclasses at all, if their interface is identical?

```python
class Request():
    def __init__(self, fileObjsDict, postObjsDict, metaObjsDict) -> None:
        self.FILES = DummyObject(fileObjsDict)
        self.POST = DummyObject(postObjsDict)
        self.META = DummyObject(metaObjsDict)
```

Reply:
You're right, i keep repeating things.